lib/xandra/batch.ex

```diff
@@ -73,6 +75,24 @@ defmodule Xandra.Batch do
     add_query(batch, prepared, values)
   end
 
+  def add(%__MODULE__{} = batch, %Prepared{} = prepared, values) when is_map(values) do
+    positional_values = Enum.map(prepared.bound_columns, fn {_keyspace, _table, name, _type} ->
+      case Map.fetch(values, name) do
+        {:ok, value} -> value
+        :error -> raise "missing named parameter #{inspect(name)} for prepared query in batch"
```

Comment:
@lexmag I think we should raise here, but what? `Xandra.Batch.MissingParameter`? `ArgumentError`?